src/index.html

```diff
@@ -11,7 +11,108 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form onsubmit="onSubmit()" action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset> <legend>Personal information</legend>
+        <div>
+          <label for="surname">Surname:</label>
+          <input type="surtext" id="surname"
+          name="surname" minlength="2"
+          maxlength="35" autocomplete="off">
```

Comment:
```suggestion
          <input 
            type="surtext" 
            id="surname"
            name="surname" 
            minlength="2"
            maxlength="35" 
            autocomplete="off"
          >
```
Один атрибут - одна линия, закрывающая скобка тега на той же линии что  и открывающая